package.json

```diff
@@ -174,6 +174,7 @@
   "prettier": {
     "singleQuote": true
   },
+  "packageManager": "yarn@1.22.18+sha1.4ba7fc5c6e704fce2066ecbfb0b0d8976fe62447",
```

Comment:
sha1 hash grabbed from `npm show list yarn@1.22.19` (ref https://github.com/nodejs/corepack/issues/231)

Reply:
@erickzhao, just realized this lists `1.22.18` but the SHA is for `1.22.19` - is that going to cause a problem?